sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -193,18 +193,16 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
     checkAnswer(sql("SHOW TBLPROPERTIES parquet_tab2('`prop2Key`')"), Row("prop2Val"))
   }
 
-  test("show tblproperties for spark temporary table - AnalysisException is thrown") {
+  test("show tblproperties for spark temporary table - empty row") {
```

Comment:
can we move this test to `show-tblproperties.sql` as well?

Reply:
OK, will do.